handlers/ci.go

```diff
@@ -1,29 +1,179 @@
+	ContentTypeJSON  = "application/json"          //content-type
+	TravisAPIVersion = "Travis-API-Version"        //API version, a Mandatory header field for Travis-CI V3 APIs
+	Build            = "build"                     //Job name
+	Verify           = "verify"                    //Job name
+	Unittest         = "unittest"                  //Job name
+	Integration      = "integration"               //Job name
+	Crossbuild       = "crossbuild"                //Job name
+
+)
+
+//TravisCI `/requests` API structure response
+type TravisRespStruct struct {
+	Requests []struct {
+		Builds []struct {
+			Href              string `json:"@href"`
+			PullRequestNumber int    `json:"pull_request_number"`
+		} `json:"builds"`
+	} `json:"requests"`
 }
 
-type CircleCIResp struct {
-	BuildURL string `json:"build_url"`
+//TravisCI `/jobs` API structure response
+type TravisJobRespStruct struct {
+	Jobs []struct {
+		Type   string `json:"@type"`
+		Href   string `json:"@href"`
+		Number string `json:"number"`
+	} `json:"jobs"`
 }
 
-const (
-	CircleCIGithubURL = "https://circleci.com/api/v1.1/project/github"
-	ContentTypeJSON   = "application/json"
-)
+//Function to handle Get Jobs from TravisCI
+func GetJobIdsFromTravisBuild(BuildRefId string, Token string) []byte {
+	var Reqbody io.Reader
+
+	url := fmt.Sprintf("%s%s/jobs", TravisCIEndPoint, BuildRefId)
+	req, err := http.NewRequest(http.MethodGet, url, Reqbody)
+	if err != nil {
+		glog.Errorf("%v", err)
+	}
+	_, Body := SendHttpReqToCI(req, Token)
+	return Body
+}
 
-func (s *Server) SendToCI(info map[string]string) {
-	glog.Info("going to send test request to ci")
+//Function to handle Triggering Jobs build in TravisCI
+func TriggerJob(JobBuildId string, Token string) {
+	var Reqbody io.Reader
 
+	url := fmt.Sprintf("%s%s/restart", TravisCIEndPoint, JobBuildId)
+	req, err := http.NewRequest(http.MethodPost, url, Reqbody)
+	if err != nil {
+		glog.Errorf("%v", err)
+	}
+	StatusCode, _ := SendHttpReqToCI(req, Token)
+	if StatusCode == http.StatusAccepted {
+		glog.Info("Restart Job Build is successfully triggered !!")
+	}
 }
 
-func (s *Server) SendToTravisCI(b []byte) {
+//Function to handle send HTTP request to TravisCI
+func SendHttpReqToCI(req *http.Request, Token string) (int, []byte) {
+	client := &http.Client{}
+
+	req.Header.Set("Content-Type", ContentTypeJSON)
+	req.Header.Set("Authorization", "token "+Token)
+	req.Header.Set(TravisAPIVersion, "3")
+	//Http Do req, For getting the PR Request information for particular repo
+	resp, err := client.Do(req)
+	if err != nil {
+		glog.Errorf("HTTP Do request is failed: %v", err)
+	}
+	body, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		glog.Errorf("fail to read resp: %v", err)
+	}
+	return resp.StatusCode, body
+}
+
+//Function to handle RETEST all jobs on PR changes
+func (s *Server) SendToCIForRetestAllJobs(PRID int) {
+	glog.Info("going to send test request to circle ci")
+
+	var Reqbody io.Reader
```

Comment:
all local variables better avoid with capitalized letter at the beginning. applies to all the code throughout.